konnectivity-client/pkg/client/client.go

```diff
@@ -88,11 +88,11 @@ func (t *grpcTunnel) serve(c *grpc.ClientConn) {
 			return
 		}
 		if err != nil || pkt == nil {
-			klog.Warningf("stream read error: %v", err)
+			klog.V(4).Infof("stream read error: %v", err)
 			return
 		}
 
-		klog.V(6).Infof("[tracing] recv packet, type: %s", pkt.Type)
+		klog.V(4).Infof("[tracing] recv packet, type: %s", pkt.Type)
```

Comment:
Unusual circumstances such as DialResp not recognized should be higher priority log level than something like book keeping that we got a packet. Guessing something like this should probably be a 6.